src/DataStructures/InfluxDBClient.cs

```diff
@@ -238,19 +240,36 @@ public int Port
         /// <param name="InfluxUrl">Url for the Inflex Server, e.g. http://localhost:8086</param>
         /// <param name="UserName">User name to authenticate InflexDB</param>
         /// <param name="Password">password</param>
-        public InfluxDBClient(string InfluxUrl, string UserName, string Password)
-        {
-            this._influxUrl = InfluxUrl;
-            this._influxDBUserName = UserName;
-            this._influxDBPassword = Password;
-
-            _client = new HttpClient();
-           //if (!(String.IsNullOrWhiteSpace(InfluxDBUserName) && String.IsNullOrWhiteSpace(InfluxDBPassword)))
-           //    _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic",
-           //        Convert.ToBase64String(Encoding.GetEncoding("iso-8859-1").GetBytes($"{InfluxDBUserName}:{InfluxDBPassword}")));
-            _client.DefaultRequestHeaders.ConnectionClose = false;
+        public InfluxDBClient(string InfluxUrl, string UserName, string Password) :
+            this(InfluxUrl, UserName, Password, null)
+        {            
         }
 
+        public InfluxDBClient(string InfluxUrl, string UserName, string Password, HttpClientHandler httpClientHandler)
+        {
+            try
+            {                               
+                this._influxUrl = InfluxUrl;
+                this._influxDBUserName = UserName;
+                this._influxDBPassword = Password;                
+
+                if (httpClientHandler != null)                               
+                    _client = new HttpClient(httpClientHandler, true);                
+                else
+                    _client = new HttpClient();
+
+                if (!(String.IsNullOrWhiteSpace(InfluxDBUserName) && String.IsNullOrWhiteSpace(InfluxDBPassword)))
+                    _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic",
+                        Convert.ToBase64String(Encoding.GetEncoding("iso-8859-1").GetBytes($"{InfluxDBUserName}:{InfluxDBPassword}")));
```

Comment:
this was commented to fix #38. Have you tested to confirm it works with non ascii characters in password? Correct way is to use UTF-8, but I am not sure if Influx supports UTF-8 Authorization header.